jenkins/jenkins2-ha-agents.yaml

```diff
@@ -2370,7 +2387,28 @@ Resources:
               - 'ecr:InitiateLayerUpload'
               - 'ecr:PutImage'
               - 'ecr:UploadLayerPart'
-
+  DeploymentDockerRegistry:
```

Comment:
@housni you mentioned on Slack that there may need to be a different ECR for each github repo.  Why is that?  I think I'm still missing a piece of the puzzle here.